config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -21,11 +21,11 @@
 base="$(dirname $0)"
 
 # The latest stable Kubernetes version for testing alpha repos
-latest_stable_k8s_version="1.16.2"
-latest_stable_k8s_minor_version="1.16"
+latest_stable_k8s_version="1.17.0"
+latest_stable_k8s_minor_version="1.17"
 
 # We need this image because it has Docker in Docker and go.
-dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20191204-a3f3e16-master"
+dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20191209-15bc109-master"
```

Comment:
Not sure the process for this, but I just grabbed the latest tag from gcr.io/k8s-testimages/kubekins-e2e